firmware/user/modes/mode_flight.c

```diff
@@ -912,17 +1149,31 @@ static bool ICACHE_FLASH_ATTR flightRender(void)
 #endif
     }
 #ifdef EMU
-	uint32_t stop = 0;
+    uint32_t stop = 0;
 #else
-	uint32_t stop = xthal_get_ccount();
+    uint32_t stop = xthal_get_ccount();
 #endif
 
-	{
-		char framesStr[32] = {0};
-		ets_snprintf(framesStr, sizeof(framesStr), "%02x %dus", tflight->buttonState, (stop-start)/160);
-		fillDisplayArea( 0, 0, 50, 7, BLACK );
-		plotText(1, 1, framesStr, TOM_THUMB, WHITE);
-	}
+
+    if( flight->mode == FLIGHT_GAME )
+    {
+        char framesStr[32] = {0};
+        //ets_snprintf(framesStr, sizeof(framesStr), "%02x %dus", tflight->buttonState, (stop-start)/160);
+        ets_snprintf(framesStr, sizeof(framesStr), "%d %d %d", tflight->ondonut, tflight->beans, tflight->timer );
+
+        plotText(1, 1, framesStr, TOM_THUMB, WHITE);
```

Comment:
Maybe try this with IBM_VGA_8 instead for readability? I would also recommend blacking out the area behind the text first, including at least a 1px border around the text so that the 3d model pixels aren't directly adjacent to the text.